noir/compiler/noirc_frontend/src/lib.rs

```diff
@@ -74,6 +75,15 @@ pub mod macros_api {
             crate_id: &CrateId,
             context: &HirContext,
         ) -> Result<SortedModule, (MacroError, FileId)>;
+
+        fn process_unresolved_traits_impls(
```

Comment:
Can we add an issue about generalizing to a `process_collected_crate` method? This is non-blocking though and can be done later

Reply:
Created https://github.com/AztecProtocol/aztec-packages/issues/4653